### ISSUE TITLE
python-tools initiates several python instances on Windows

### ISSUE THREAD

**Issue by mostaphaRoudsari** [MichaelAquilina/python-tools#39]
After updating Atom I get the warning that python-tool is unable to find python executable (#36 and #38). Adding path in settings solves path issue but then it initiates several python instances at the same time. I had to disable the package for now.

![image](https://cloud.githubusercontent.com/assets/2915573/14676673/83e047f2-06dc-11e6-8f53-43acf3acc3f1.png)

![image](https://cloud.githubusercontent.com/assets/2915573/14676533/db672e74-06db-11e6-8d28-4478a33b477e.png)

**Comment by trunneml** [comment]
On my windows system there is only one Python instance. But that produces a high CPU load (1 complete Core). That makes the plugin nearly uses less on my laptop (heat/fan/battery).

**Comment by liukai826** [comment]
It  also occured on my windows, Plz @me  when figure it out

**Comment by mostaphaRoudsari** [comment]
I took a closer look and it seems not to be an issue with python-tools but actually autocomplete-python package. I'm closing this issue and will open a new one in [autocomplete-python](https://github.com/sadovnychyi/autocomplete-python/issues). Last time that I checked I just read the command line for the first one.

![autocomplete_python](https://cloud.githubusercontent.com/assets/2915573/14839941/1897e20e-0bfd-11e6-83bf-1614e316ad66.PNG)
